string robot_name
geometry_msgs/PoseStamped[] locations_to_visit 